### ISSUE TITLE
[Feature request] Option to disable social buttons

### ISSUE THREAD

**Issue by TehPeGaSuS** [MercanoGlobal/SNIP#10]
I may be overlooking something but I don't see a way to disable the social (Facebook, X, WhatsApp, Email) buttons.

**Comment by MercanoGlobal** [comment]
This is a theme feature and can be easily removed just by deleting/commenting the social code in `themes/name_of_the_theme/views/view/view.php`:
```
	<div class="detail" style="margin-top:5px;">
		<!-- AddToAny BEGIN -->
		<span class="a2a_kit a2a_kit_size_32 a2a_default_style">
			<a class="a2a_button_facebook"></a>
			<a class="a2a_button_x"></a>
			<a class="a2a_button_whatsapp"></a>
			<a class="a2a_button_email"></a>
		</span>
		<script async src="https://static.addtoany.com/menu/page.js"></script>
		<!-- AddToAny END -->
	</div>
```
It's not worth the hassle of implementing a config feature just for this, as the config file is already huge, and it will become even more confusing for beginner users.

**Comment by TehPeGaSuS** [comment]
I strongly disagree, since I don't understand PHP (like lots of other people) and I don't want to mess up things.

But I do understand the why of not wanting to implement the feature.

**Comment by MercanoGlobal** [comment]
I don't want to add a setting for this, because this isn't a core feature, but a theme one.
Basically, you can call it a design choice.

Also, there have been countless times when we wanted to quickly share a piece of code on our WhatsApp groups, and it was just a hassle to copy/paste the link. Now, with the share buttons, we can do that in 2 clicks.
On your website, I see you removed the share buttons. If so, then why don't you also remove the URL and Embed, which are also related to sharing a paste? See what I mean?
![ssshare](https://github.com/user-attachments/assets/3ad6c1dc-296c-4e37-a079-72124eb244b0)

**Comment by TehPeGaSuS** [comment]
I still want people to share the URL, since this is a test instance for a production instance that I have.
I just don't need Facebook and all those bells and whistles.
A `Copy URL` button, for me, would be more suitable.

**Comment by MercanoGlobal** [comment]
Well, from what I've seen, only us and Pastebin.com have this feature.
![pbin](https://github.com/user-attachments/assets/7cdad3d8-d52e-4bcb-8c7f-c12b19a435a1)

Our implementation being a bit better 😁 
Quick sharing is part of the future. Many developers that use such services, share code snippets on Twitter all the time.

**Comment by TehPeGaSuS** [comment]
Yes, I know.

But I'm providing a pastebin for IRC.

We only need, at most, a `Copy URL` button (which currently we don't have)
